.sdkmanrc

```diff
@@ -0,0 +1,3 @@
+# Enable auto-env through the sdkman_auto_env config
```

Comment:
did you mean to commit this?

Reply:
Yes it's been hanging around for ages in my  folder and I think it's useful